### DISCUSSION TITLE
grbl Twotrees Totem S laser engraver (v.4.4.1) not moving

### DISCUSSION THREAD

**Discussion by Merckymerc** [arkypita/LaserGRBL#1559] [Q&A]
Just purchased this laser engraver. Finally got everything installed correctly (I believe). Driver, config file, baud rate, port all working. Custom buttons come up. Lights are on on the actual laser. It seems to "connect" correctly (it says connected).

But the jog buttons create no movement. If an image is loaded, laser head does not move at all when I start the job.

Help please!

**Comment by gmmanonymus111** [comment]
I'm curious, did you unlock your machine before trying to move? GRBL by default, resets itself on powerup for safety reasons. To unlock your machine and enable movement, you should click the `Padlock` button.
![image](https://user-images.githubusercontent.com/9658243/131036541-76db3f76-ca1f-4600-9e5f-bb76013930e8.png)

**Comment by Merckymerc** [comment]
Yes, I clicked on the unlock button. Still nothing.

I tried switching some of the wiring on the motors, also to no avail.

Thanks for your assistance, though. I appreciate it.

**Comment by twotrees99** [comment]
Hello, dear buyer.
I am honored to answer your question. This is a website sent to me by a customer who encountered the same problem.

Our engraving can only supply power to the motherboard through the USB cable. Even if the power cable is not connected, as long as the USB cable is connected, its motherboard and laser will be bright.

However, when there is only USB power supply, only 5V is not enough to control the movement of the machine.

So I now need you to check whether the adapter is normal and whether it can supply power normally.
(Can test whether it can power other electrical appliances)

I don’t know where you bought our machine or whether your problem has been solved.
If you have any questions, please contact us via email. cm@twotrees3d.com
sincere greetings!

**Reply by arkypita** [reply]
@twotrees99  I appreciate that twotrees3d give help to her customer.
Arkypita, LaserGRBL developer

**Comment by twotrees99** [comment]
Thank you for your letter.
It is my duty to help customers.
